webapp/client/src/components/InfoBox.js

```diff
@@ -48,6 +48,10 @@ const Figure = styled.figure`
   img {
     max-width: 500px;
     height: auto;
+
+    @media (max-width: 425px) {
+      max-width: 100%;
+    }
   }
```

Comment:
We are going to optimize the image in another subtask, but leave it as it is right now.